components/script/dom/eventtarget.rs

```diff
@@ -225,7 +228,11 @@ impl CompiledEventListener {
 /// A listener in a collection of event listeners.
 struct EventListenerEntry {
     phase: ListenerPhase,
-    listener: EventListenerType
+    listener: EventListenerType,
+    capture: bool,
```

Comment:
This new field is useless for now, given `phase` already covers it, please remove.